advocacy_docs/partner_docs/CommVaultGuide/05-UsingCommvault.mdx

```diff
@@ -0,0 +1,188 @@
+---
+title: 'Using Commvault Backup & Recovery'
+description: 'Walkthroughs of multiple Commvault Backup & Recovery usage scenarios'
+---
+
+How to backups and restore from EDB Postgres Advanced Server using Commvault Backup & Recovery.
+
+## Using Commvault Backup & Recovery
+
+Commvault provides the two methods of taking the backup from EDB Postgres Advanced Server and restoring it.
```

Comment:
Commvault provides the two methods 
chang
Commvault provides two methods ....       Don't need "the'